### PR TITLE
Line comments instead of block comments 10

### PR DESCRIPTION
**Pull request by GinoGiotto**
Continuation of https://github.com/metamath/metamath-exe/pull/138

This PR covers the `mmpfas.c` file. Only the header, a few inline comments and `/*E*/` comments are left out.

### CODE REVIEW

**Comment by GinoGiotto** [comment]
There are currently no merge conflicts, but since comments are widespread throughout the code, these pending PRs might create complications for future commits. @digama0

File: src/mmpfas.c [hunk 1]
```diff
@@ -1021,128 +1025,128 @@ vstring getIndepKnownSteps(long proofStmt, long refStep)
...
+// The rawTargetProof may be uncompressed or compressed.
+// Note: The caller must deallocate the returned nmbrString.
 nmbrString *expandProof(
-    const nmbrString *rawTargetProof, /* May be compressed or uncompressed */
-    long sourceStmtNum   /* The statement whose proof will be expanded */
-    /* , long targetStmtNum */) { /* The statement begin proved */
+    const nmbrString *rawTargetProof, // May be compressed or uncompressed
+    long sourceStmtNum // The statement whose proof will be expanded
+    // , long targetStmtNum
+    ) { // The statement begin proved
```
Comment (digama0) [inline src/mmpfas.c:1149]:
```suggestion
    // , long targetStmtNum // The statement being proved
    ) {
```

**Comment by GinoGiotto** [comment]
Can we review these PRs? They've been pending for months and I'd like to avoid potential merge conflicts. @digama0

File: src/mmpfas.c [hunk 2]
```diff
@@ -660,101 +661,101 @@ nmbrString *replaceStatement(long replStatemNum, long prfStep,
...
-                 so we might need a factor of 50 greater runtime for each
-                 replacement hypothesis having this situation.) */
-              /* trialStep = prfStep - 1;  old version */
+              // There are no dummy variables, so previous match
+              // was exact.  Force the trialStep loop to terminate as
+              // if nothing further was found.  (If we don't do this,
+              // there could be, say 50 more matches for "var x",
+              // so we might need a factor of 50 greater runtime for each
+              // replacement hypothesis having this situation.)
+              // trialStep = prfStep - 1;  old version
```
Comment (digama0) [inline src/mmpfas.c:701]:
```suggestion

              // trialStep = prfStep - 1;  old version
```

**Pull request merged**